tasks/synopsys-detect-task/ts/DetectUtils.ts

```diff
@@ -0,0 +1,12 @@
+// TODO verify if this parsing works for new lines (May want to separate on spaces/newlines
```

Comment:
Looks like you have a test-case for this, so it might be "done". Also, missing right parenthesis at the end of the comment.